src/emmental/logging/logging_manager.py

```diff
@@ -173,8 +175,22 @@ def write_log(self, metric_dict: Dict[str, float]) -> None:
         Args:
           metric_dict: The metric dict.
         """
+        unit_total: Union[float, int] = self.unit_total
+        if (
+            Meta.config["logging_config"]["writer_config"]["writer"] == "tensorboard"
```

Comment:
I might add a comment that tensorboard assumes the x axis value (global step) is an integer

Reply:
Good catch!